src/__tests__/transformers/Streamable.js

```diff
@@ -0,0 +1,49 @@
+import cases from 'jest-in-case';
+
+import { getHTML, shouldTransform } from '../../transformers/Streamable';
+
+jest.mock('node-fetch', () =>
```

Comment:
Check the Twitter tests to make this a bit cleaner (can be extracted in another PR)